docs/src/modules/components/AppFrame.js

```diff
@@ -146,6 +147,15 @@ class AppFrame extends React.Component {
             )}
             <div className={classes.grow} />
             <AppSearch />
+            <Tooltip id="appbar-color" title="Edit docs colors" enterDelay={300}>
+              <IconButton
+                color="inherit"
+                href="/style/color/#color-tool"
```

Comment:
Better use the next.js link. I can do it if you want.

Reply:
I tried, but it doesn't seem to respect the hash. (Links to the page, but not to the section):
```jsx
<IconButton
  color="inherit"
  component={buttonProps => (
    <Link variant="button" href="/style/color/#color-tool" {...buttonProps} />
  )}
  aria-labelledby="appbar-color"
 >
```
You have to click twice - once to go to the page, once to go to the section.